portfolio/src/main/webapp/script.js

```diff
@@ -13,8 +13,7 @@
 // limitations under the License.
 
 /**
- * Generates a URL for a random image in the images directory and adds an img
- * element with that URL to the page.
+ * Adds a random quote to the page.
```

Comment:
nit: "adds" makes me think every time this function is called, a new quote is added to the page. It's not the case, however. There will be at most one quote shown. So maybe just say "shows a random quote in the page" or something similar?